app/src/services/section.services.ts

```diff
@@ -26,10 +26,40 @@ export const saveSection = async (data: any, id: any/*, token: string*/) => {
     );
 };
 
+
+/**
+ * Create a new section for a course
+ * 
+ * @param data Input data, should be a Section interface
+ * @param id id of the section
+ * @param token token of the user, currently ignored
+ * @returns respons from the backend post request
+ */
+const createSection = async (data: any, id: any, token: string) => {
+    return await axios.post(
+      `${BACKEND_URL}/api/sections/create/${id}`,
+      data,
+      { headers: { Authorization: `Bearer ${token}` } }
+    );
+}
+
+const deleteSection = async (id: any, token: string) => {
+    console.log("Deleted section with id: " + id)
+    return await axios.delete(
+        `${BACKEND_URL}/api/sections/delete/${id}`,
```

Comment:
Do not have actions in endpoints

should be api/sections/:id

Reply:
Thanks for the reminder, we have fixed it